client/components/dashboard/dashboard-service.js

```diff
@@ -421,12 +484,14 @@ DashboardService.prototype.selectWidget = function(widget, container) {
     this.scrollWidgetIntoView(widget);
   });
 
-  params = {widget: undefined, container: undefined};
+  if (opt_supressStateChange !== true) {
```

Comment:
The intent was the user having to explicitly pass "true" as a parameter to suppress, but I agree it's better to leave the parameter with javascript default boolean logic.